src/main/java/com/lberkholtz/advancedava/week3/service/StockServiceFactory.java

```diff
@@ -1,13 +1,17 @@
+/**
+ * @author Laura Berkholtz
+ */
+
 package com.lberkholtz.advancedava.week3.service;
 
 /**
  * Returns a concrete implementation of the
- * interface depending on
+ * interface depending on the parameter passed
  */
 public class StockServiceFactory{
     /**
      * Just coding for BasicStockService now, can add more types of StockService in the future.
-     * @param stockServiceType
+     * @param stockServiceType what type of stock service is needed. For now we just have Basic
```

Comment:
recall from the chat, we don't want the client code (code calling our code) to know about the concrete types, just the interfaces. If this is not clear to you, set up a 1:1 with me and we can review.

Reply:
I am going to have to set up a 1 on 1 to make sure I understand this. I think I'm missing something.